tabular/src/autogluon/tabular/models/tab_transformer/hyperparameters/searchspaces.py

```diff
@@ -0,0 +1,10 @@
+from autogluon.core import Real
+
+# TODO: I don't see a need to split search space by problem type.
+#  Is there a reason to at this point?
```

Comment:
probably don't need to split search space by problem type at this time, can leave comment as reminder to consider splitting in the future.

Reply:
Added a comment for potential future work.